shardingsphere-test/shardingsphere-pipeline-test/src/test/java/org/apache/shardingsphere/data/pipeline/core/lock/PipelineSimpleLockTest.java

```diff
@@ -0,0 +1,86 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.data.pipeline.core.lock;
+
+import org.apache.shardingsphere.data.pipeline.core.context.PipelineContext;
+import org.apache.shardingsphere.infra.instance.InstanceContext;
+import org.apache.shardingsphere.mode.manager.ContextManager;
+import org.apache.shardingsphere.mode.manager.cluster.coordinator.lock.service.LockNode;
+import org.apache.shardingsphere.mode.manager.cluster.coordinator.lock.service.LockRegistryService;
+import org.apache.shardingsphere.mode.metadata.MetaDataContexts;
+import org.apache.shardingsphere.mode.metadata.persist.MetaDataPersistService;
+import org.apache.shardingsphere.mode.repository.cluster.ClusterPersistRepository;
+import org.apache.shardingsphere.transaction.context.TransactionContexts;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.lang.reflect.Field;
+import java.util.concurrent.TimeUnit;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public final class PipelineSimpleLockTest {
+
+    @Mock
+    private ClusterPersistRepository clusterPersistRepository;
+
+    @Mock
+    private MetaDataPersistService metaDataPersistService;
+
+    @Mock
+    private MetaDataContexts metaDataContexts;
+
+    private ContextManager contextManager;
+
+    private LockRegistryService lockService;
+
+    private PipelineSimpleLock pipelineSimpleLock;
+
+    @Before
+    public void setUp() throws ReflectiveOperationException {
+        metaDataPersistService = new MetaDataPersistService(clusterPersistRepository);
+        metaDataContexts = new MetaDataContexts(metaDataPersistService);
+        contextManager = new ContextManager();
+        contextManager.init(metaDataContexts, mock(TransactionContexts.class), mock(InstanceContext.class));
+        PipelineContext.initContextManager(contextManager);
+        lockService = new LockRegistryService(clusterPersistRepository);
+        Field field = lockService.getClass().getDeclaredField("repository");
+        field.setAccessible(true);
+        field.set(lockService, clusterPersistRepository);
+        pipelineSimpleLock = PipelineSimpleLock.getInstance();
+    }
+
+    @Test
+    public void assertTryLock() {
+        pipelineSimpleLock.tryLock("test", 50L);
+        lockService.tryLock("test", 50L);
+        verify(clusterPersistRepository).tryLock(LockNode.getLockNodePath("test"), 50L, TimeUnit.MILLISECONDS);
+    }
```

Comment:
`lockService.tryLock("test", 50L);` is not necessary, since `lockService` is encapsulated in `PipelineSimpleLock`, we could just test `pipelineSimpleLock.tryLock`.